cmd/podman/common.go

```diff
@@ -402,3 +403,32 @@ var createFlags = []cli.Flag{
 		Usage: "Working `directory inside the container",
 	},
 }
+
+func initOptions(createConfig *cc.CreateConfig, runtime *libpod.Runtime) ([]libpod.CtrCreateOption, error) {
+	useImageVolumes := createConfig.ImageVolumeType == "bind"
+
+	options, err := createConfig.GetContainerCreateOptions()
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to parse new container options")
+	}
+	// Gather up the options for NewContainer which consist of With... funcs
+	options = append(options, libpod.WithRootFSFromImage(createConfig.ImageID, createConfig.Image, useImageVolumes))
```

Comment:
Can we just put all of this in `GetContainerCreateOptions()` instead of making a new function?

Reply:
Great point, and this led me to clean up varlink also.